### PR TITLE
Implement Spanish language

### PR DESCRIPTION
**Pull request by pablorodriper**
Hi,

I've implemented Spanish language support for your module. 

It works from 0 to 999 999 999 999 like the other languages, but ord2card() is not implemented.

Hope it helps.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -108,11 +122,26 @@ English:
...
 
+Spanish (ordinals not supported):
+
+.. code-block:: python
+
+    >>> from text_to_num import alpha2digit
+    
+    >>> text = "Compramos veinticinco vacas, doce gallinas y ciento veinticinco coma cuarenta kg de patatas."
+    >>> alpha2digit(text, "es")
+    'Compramos 25 vacas, 12 gallinas y 125.40 kg of potatoes.'
```
Comment (rtxm) [inline README.rst:139]: of potatoes? :wink:

**Comment by pablorodriper** [comment]
> Thank you very much for your contribution!
> I don't know Spanish, can you explain what is the difficulty with ordinals?

It's not very complicated, but we haven't got a unique suffix so it is like writing the cardinal number with other words. You can understand all the possibilities with the next picture:
https://es-static.z-dn.net/files/d82/d4d8c3ae63d4f2313b45fc5f13868ea7.png

**Comment by rtxm** [comment]
I see, better let them in full letters then!

File: text_to_num/transforms.py [hunk 2]
```diff
@@ -74,6 +74,8 @@ def alpha2digit(
     if you prefer to get « moins 2 » instead of « -2 ».
     """
     language = LANG[lang]
+    if lang is "es":
```
Comment (rtxm) [inline text_to_num/transforms.py:77]: `is` raises a warning with literals; you should use `==` here.

**Comment by rtxm** [comment]
I will merge as soon as you fixes the two little nits. Thank you.

**Pull request merged**